pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -135,4 +132,100 @@ public void run() {
       }
     }
   }
+
+  /**
+   * blocks dump sub-command
+   *
+   * <p>Dump a specific block from storage
+   */
+  @Command(
+      name = "dump",
+      description = "This command dump a specific block from storage",
+      mixinStandardHelpOptions = true)
+  static class DumpSubCommand implements Runnable {
+    @SuppressWarnings("unused")
+    @ParentCommand
+    private BlocksSubCommand parentCommand; // Picocli injects reference to parent command
+
+    @Option(
+        names = "--number",
```

Comment:
```suggestion
        names = "--start-block",
```

Reply:
I just made the requested changes here https://github.com/PegaSysEng/pantheon/pull/1641